library/core/resource_loader/src/main/java/org/quiltmc/qsl/resource/loader/impl/ResourceLoaderImpl.java

```diff
@@ -254,19 +252,23 @@ public static void appendResourcesFromGroup(NamespaceResourceManagerAccessor man
 
 	/* Built-in resource packs */
 
+	public static Text getBuiltinPackDisplayNameFromId(Identifier id) {
+		return new LiteralText(id.getNamespace() + "/" + id.getPath());
```

Comment:
Maybe a `TranslatableText` here?

Reply:
I don't see an interest into making it translatable? This name is a raw name, if you want to translate it you should use the display name parameter.